cumulusci/tasks/marketing_cloud/base.py

```diff
@@ -15,3 +16,18 @@ class BaseMarketingCloudTask(BaseTask):
     def _init_task(self):
         super()._init_task()
         self.mc_config = self.project_config.keychain.get_service("marketing_cloud")
+
+    def _check_response(self, response):
+        """Make sure the response indicates success."""
```

Comment:
If this is SOAP API specific maybe this should be `_check_soap_response()` instead?

Reply:
Good call; done